usaspending_api/api_contracts/contracts/agency/federal_obligations/FederalAccount.md

```diff
@@ -0,0 +1,74 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Agency Profile
+
+These endpoints are used to power USAspending.gov's agency profile pages. This data can be used to better understand the different ways that a specific agency spends money.
+
+### Get Major Object Classes [GET]
```

Comment:
It looks like this is actually getting Minor Object Classes, since `major_object_class_code` is one of the params.